### ISSUE TITLE
Markdown content export/import

### ISSUE THREAD

**Issue by kyrylolvov** [toeverything/blocksuite#6291]
Is there a way to get content of the page in markdown format without downloading the file, and is there a way to set page to markdown content without uploading the file?

**Comment by doodlewind** [comment]
Yep, should be fine for changing the return value of markdown exporting method.
